actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -77,7 +95,7 @@ class ActionCable.Connection
       {identifier, message, type} = JSON.parse(event.data)
       switch type
         when message_types.welcome
-          @monitor.recordConnect()
+          break
```

Comment:
You sold me on the welcome message in https://github.com/rails/rails/pull/23976#issuecomment-190655306 :grin: 
> I also like the idea that, especially when there is potentially authentication/the possibility that the server will reject the connection, the server sends a definitive message to the client saying you are connected: hence welcome.